certbot/compat.py

```diff
@@ -0,0 +1,108 @@
+"""
```

Comment:
I'm sorry I missed this in my previous review, but can we document the parameter and return types of the functions in this module? How we do this for Certbot is documented at https://certbot.eff.org/docs/contributing.html#coding-style.

There are a lot of examples in our code, but don't hesitate to ping me if you have any questions.

Reply:
Done.